src/containers/VDiskPage/VDiskPage.tsx

```diff
@@ -70,27 +71,31 @@ export function VDiskPage() {
 
     const handleEvictVDisk = async (isRetry?: boolean) => {
         if (vDiskIdParamsDefined) {
-            return (
-                newDiskApiAvailable ? window.api.vdisk.evictVDisk : window.api.tablets.evictVDiskOld
-            )({
```

Comment:
In such construction `this` is lost, so `this.post` inside functions doesn't work.

How it should be done in such case (adding `bind`):
```js
newDiskApiAvailable ? 
    window.api.vdisk.evictVDisk.bind(window.api.vdisk) :
    window.api.tablets.evictVDiskOld.bind(window.api.tablets)
```